API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
+**LessonPlan** | *:*  
```

Comment:
I think a `through` table makes more sense here though because that's what it's meant for - storing extra data about a relation. If the relation is between a module and an activity, the extra data would be the order. I believe we did a similar thing in the science api as well so maybe that could be a good reference for implementing this